### ISSUE TITLE
Request for @crowdstrike to port project

### ISSUE THREAD

**Issue by simonsigre** [simonsigre/crowdstrike_falcon-ipaddresses#1]
This project is to maintain the list of CrowdStrike Falcon IP addressing for use in dynamic objects/lists on firewalls .. currently I manually check documentation on a sporadic basis and update accordingly.
I believe the project should be 'formalised' and ported over to be a @crowdstrike project (even if the community of partners/customers helps maintain it) .

Unsure who to request this via so going to just tag as many `cs` folks as I can hopefully someone knows who I should talk to.

@alexgraul
@cgutman
@csscott
@deckarep
@dmitria
@jiminoc
@john-kurkowski
@mr-burnse
@pwinkler
@rogerclermont
@schleppy
@skrasser
@vitch
@willthetanman

**Comment by MDKza** [comment]
Where are you pulling this data from exactly? Starting my own EDL manager project, will find a way to add this as one of the lists

**Comment by simonsigre** [comment]
@TyronFourie  Manual post process .... hasn't been updated on a while ...your plan is something similar to this https://docs.paloaltonetworks.com/resources/edl-hosting-service ?

You can actually request PAN add new services; our team had some of the regional Okta stuff added .

I'd happily contribute ... also consider this 

https://github.com/MISP/misp-warninglists

**Comment by MDKza** [comment]
Yes like that but also self managed lists by users. Does your gmail address work? Will pop you a mail with the BETA URL.
